### PR TITLE
Omit SafeAreaView for old iOS versions

### PR DESCRIPTION
**Pull request by kirillpisarev**
#181

### CODE REVIEW

File: DropdownAlert.js [hunk 1]
```diff
@@ -448,6 +448,7 @@ export default class DropdownAlert extends Component {
   }
   render() {
     const { isOpen, type } = this.state;
+    const Body = Platform.OS === 'ios' && parseInt(Platform.Version, 10) < 11 ? View : SafeAreaView;
```
Comment (testshallpass) [inline DropdownAlert.js]: Call it: `ContentView`

Comment (testshallpass) [reply DropdownAlert.js]: Actually, make changes:

```
// put in constants.js
const IS_IOS_BELOW_11 = IS_IOS && parseInt(Platform.Version, 10) < 11;
// ....
import { IS_IOS_BELOW_11 } from from './constants';
const ContentView = IS_IOS_BELOW_11 ? View : SafeAreaView;
```

Comment (kirillpisarev) [reply DropdownAlert.js]: Refactored

File: constants.js [hunk 2]
```diff
@@ -8,6 +8,7 @@ const HEIGHT = WINDOW.height;
 const WIDTH = WINDOW.width;
 const IS_IOS = Platform.OS == 'ios';
 const IS_ANDROID = Platform.OS == 'android';
+const IS_IOS_BELOW_11 = IS_IOS && parseInt(Platform.Version, 10) < 11;
```
Comment (testshallpass) [inline constants.js]: you need to add it to the `module.exports`

File: DropdownAlert.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import React, { Component } from 'react';
-import { StyleSheet, SafeAreaView, View, TouchableOpacity, Animated, StatusBar, PanResponder } from 'react-native';
+import { StyleSheet, SafeAreaView, View, TouchableOpacity, Animated, StatusBar, PanResponder, Platform } from 'react-native';
```
Comment (testshallpass) [inline DropdownAlert.js]: I missed this; remove Platform.

**Comment by kirillpisarev** [comment]
Could it be merged?

**Pull request merged**